Sources/OpenAI/OpenAI+ChatCompletion.swift

```diff
@@ -63,22 +63,25 @@ public extension OpenAI {
             self.parallel_tool_calls = parallel_tool_calls
         }
 
-        public func completion(response: OpenAI.ChatCompletionResponse) throws -> ChatCompletionRequest? { // This needs tests badly
+        public func completion(response: OpenAI.ChatCompletionResponse, toolCallHandler: @escaping ([OpenAI.Message.ToolCall]) -> [String]) throws -> ChatCompletionRequest? { // This needs tests badly
```

Comment:
The toolCallHandler method should be added as a property on the ChatCompletionRequest.